performance-tests/src/test/resources/application.conf

```diff
@@ -5,7 +5,7 @@
 # The default url for every service.
 # For example
 
-baseUrl = "http://localhost:3000"
+baseUrl = "https://staging.hwf.dsd.io/"
```

Comment:
May I suggest to keep the `http://localhost:3000` and just use the `APP_URL` env variable to set the desired URL to run the tests against?

Reply:
yeah I did think the same so I'll remove this change.
Thanks Leo